bundles/org.openhab.binding.lgthinq/src/main/java/org/openhab/binding/lgthinq/internal/LGThinqBindingConstants.java

```diff
@@ -12,6 +12,8 @@
  */
 package org.openhab.binding.lgthinq.internal;
 
+import static org.openhab.binding.lgthinq.internal.handler.LGThinqBridgeHandler.THING_TYPE_BRIDGE;
+
```

Comment:
You did not understand me. You define the THING_TYPE_BRIDGE in LGThinqBindingConstants, not import it

Reply:
Sorry, I skipped this one.